scripts/import-osm

```diff
@@ -54,6 +59,7 @@ ${OSM2PGROUTING} -f /tmp/$COUNTRY.osm -c $(dirname "${BASH_SOURCE}")/mapconfig.x
 echo "Populating ways_nodes table"
 psql -d $POSTGRES_DB -U $POSTGRES_USER -h $POSTGRES_HOST << SQL_SCRIPT
 SELECT populate_ways_nodes();
+SELECT connect_isolated_segments(5000);
```

Comment:
Is the function idempotent? If so, it might make sense to add it as a migration, so we don't forget running it manually on dev, stg or prod.